src/zck.c

```diff
@@ -170,7 +172,8 @@ int main (int argc, char *argv[]) {
     char *base_name = NULL;
     char *out_name = NULL;
     if(arguments.output == NULL) {
-        base_name = basename(arguments.args[0]);
+        // base_name = basename(arguments.args[0]);
+        base_name = arguments.args[0];
```

Comment:
This is changing how the utility works.  Please revert these two lines.

Reply:
Yes, fixed.